packages/interactor/src/constructor.ts

```diff
@@ -38,6 +38,9 @@ export function findElements<E extends Element>(parentElement: Element, interact
 }
 
 function findMatches(parentElement: Element, interactor: InteractorOptions<any, any, any>): Match<Element, any>[] {
+  if (!interactor.name) {
+    throw new Error('An interactor used for this test was not given a label. Please provide a name for your interactor:\n\tHTML.extend(\'my interactor\') || createInteractor(\'my interactor\')');
```

Comment:
It's possible to use interactors outside of testing because they are a general automation tool. For example, @jgnieuwhof suggested that they could be used as screen scrapers, or any other area that calls for application automation. Can we formulate an error message that includes those too? E.g "one of your interactors was created without a name" or some variation on that?

Reply:
Changed it to `One of your interactors was created without a name. Please provide a label for your interactor: ...`